pallets/marketplace/src/lib.rs

```diff
@@ -83,18 +83,18 @@ pub mod pallet {
 	pub(super) type Offers<T: Config> = StorageDoubleMap<
 		_,
 		Blake2_128Concat,
-		(T::NftClassId, T::NftInstanceId),
+		(T::NftCollectionId, T::NftItemId),
 		Blake2_128Concat,
 		T::AccountId,
 		OfferOf<T>,
 		OptionQuery,
 	>;
 
 	#[pallet::storage]
-	#[pallet::getter(fn marketplace_instances)]
+	#[pallet::getter(fn marketplace_items)]
 	/// Stores Marketplace info
-	pub type MarketplaceInstances<T: Config> =
-		StorageDoubleMap<_, Twox64Concat, T::NftClassId, Twox64Concat, T::NftInstanceId, RoyaltyOf<T>>;
+	pub type MarketplaceItems<T: Config> =
```

Comment:
this probably requires storage migration, if there is some instance in the old storage before

Reply:
Yes, I have implemented storage migrations for NFT and Marketplace. JFYI, the migration in the Uniques pallet is for something different and this renaming is handled differently, for more info check this convo https://github.com/paritytech/substrate/pull/11389#discussion_r869490351